tests/unit/lib/DynamicArray_unittest.cc

```diff
@@ -251,10 +252,44 @@ TEST_F(CDynamicArrayFixture, append_array_bool)
 	EXPECT_EQ(wrapper_array_b->get_element(2), true);
 }
 
-TEST_F(CDynamicArrayFixture, serializable_array)
+TYPED_TEST(CDynamicArrayTest, serializable_array)
```

Comment:
In general, I would name the test after the function I test, here `save_serializable`